### PR TITLE
Add SilverBlue InstallClass

### PR DESCRIPTION
**Pull request by jkonecny12**
This may change based on the variant string and product image.

### CODE REVIEW

File: pyanaconda/installclasses/fedora_silverblue.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from pyanaconda.product import productVariant
+
+__all__ = ["FedoraSilverBlueInstallClass"]
+
+
+class FedoraSilverBlueInstallClass(FedoraWorkstationInstallClass):
+    name = "Fedora SilverBlue"
+    stylesheet = "/usr/share/anaconda/pixmaps/workstation/fedora-workstation.css"
+
+    if productVariant != "SilverBlue":
```
Comment (poncovka) [inline pyanaconda/installclasses/fedora_silverblue.py]: The name seems to be `Silverblue`.

Comment (jkonecny12) [reply pyanaconda/installclasses/fedora_silverblue.py]: Where did you find that?

Comment (poncovka) [reply pyanaconda/installclasses/fedora_silverblue.py]: Everywhere: https://github.com/rhinstaller/anaconda/issues/1505, https://pagure.io/teamsilverblue/issue/20, https://teamsilverblue.org/, https://mojefedora.cz/silverblue-atomic-workstation-pod-novou-znackou/

Comment (jkonecny12) [reply pyanaconda/installclasses/fedora_silverblue.py]: Heh, OK :)

**Comment by jkonecny12** [comment]
**UPDATED**
* Fixed name

File: pyanaconda/installclasses/fedora_silverblue.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+__all__ = ["FedoraSilverblueInstallClass"]
+
+
+class FedoraSilverblueInstallClass(FedoraWorkstationInstallClass):
+    name = "Fedora Silverblue"
+    stylesheet = "/usr/share/anaconda/pixmaps/workstation/fedora-workstation.css"
+
+    if productVariant != "Silverblue":
+        hidden = True
```
[the remark this answers is not in the document]
Comment (jkonecny12) [reply pyanaconda/installclasses/fedora_silverblue.py]: Damn I hate class level implementation. This shouldn't exists... Why this isn't property or something which we can overload?

**Comment by jkonecny12** [comment]
**UPDATED**
Silverblue installclass doesn't use inheritance anymore.

**Comment by jkonecny12** [comment]
jenkins, test this please

**Pull request merged**